tests/test_empty_value_handling.py

```
@@ -0,0 +1,349 @@
+            port=OB_PORT,
+            tenant=OB_TENANT,
+            database=OB_DATABASE,
+            user=OB_USER,
+            password=OB_PASSWORD
+        )
+        
+        # Test connection
+        try:
+            result = client._server._execute("SELECT 1 as test")
+            assert result and result[0].get("test", 1) == 1
+        except Exception as exc:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)